helper/m4/configure.m4

```diff
@@ -8,12 +8,11 @@ AC_ARG_ENABLE([test-helper],
 AM_CONDITIONAL([test_helper], [test x$test_helper = xyes ])
 
 ##########################################################################
-# Enable/disable helper-ext
-# platform specific non portable extensions
+# Enable/disable Linux helpers
 ##########################################################################
-helper_linux=no
+helper_linux=yes
 AC_ARG_ENABLE([helper-linux],
-	[  --enable-helper-linux	build helper platform extensions (not portable)],
+	[  --enable-helper-linux   enable Linux helpers],
 	[if test "x$enableval" = "xyes"; then
```

Comment:
Shouldn't we add a test now checking if $enableval is "no" here?

Reply:
corrected in v2